website/docs/r/cloud_subnet.html.markdown

```diff
@@ -39,8 +39,6 @@ Manages ACI Cloud Subnet
 * `scope` - (Optional) The domain applicable to the capability. Allowed values are "public", "private" and "shared". Default is "private".
```

Comment:
Scope is a list now but this documentation has not been updated.

Reply:
Updated the docs.